packages/devextreme/js/__internal/scheduler/workspaces/m_work_space.ts

```diff
@@ -29,6 +29,7 @@ import pointerEvents from '@js/events/pointer';
 import { addNamespace, isMouseEvent } from '@js/events/utils/index';
 import messageLocalization from '@js/localization/message';
 import { getMemoizeScrollTo } from '@js/renovation/ui/common/utils/scroll/getMemoizeScrollTo';
+import { ViewType } from '@js/renovation/ui/scheduler/types';
```

Comment:
Why do use import from renovation code? If you want using typing, may be take types from _packages/devextreme/js/ui/scheduler.d.ts_?

Reply:
I used the type from the renovation (R1) because the code that used this type passed a variable to function to the renovated (R1) code.